engine/src/test/java/org/entando/entando/web/system/ReloadConfigurationControllerTest.java

```diff
@@ -10,25 +10,27 @@
 
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
 
 public class ReloadConfigurationControllerTest extends AbstractControllerIntegrationTest {
 
     @Autowired
     @InjectMocks
     private ReloadConfigurationController controller;
 
-
     @Test
-    public void should_execute_reload() throws Exception {
-
+    public void should_execute_reload_and_have_headers() throws Exception {
 
         UserDetails user = new OAuth2TestUtils.UserBuilder("jack_bauer", "0x24").grantedToRoleAdmin().build();
         String accessToken = mockOAuthInterceptor(user);
         ResultActions result = mockMvc
-                                      .perform(post("/reloadConfiguration")
-                                                                           .header("Authorization", "Bearer " + accessToken));
+                .perform(post("/reloadConfiguration")
+                        .header("Authorization", "Bearer " + accessToken));
         result.andExpect(status().isOk());
+        result.andExpect(header().string("Access-Control-Allow-Origin", "*"));
```

Comment:
Please  add a comment about what is being tested here so that if this breaks we know why it is here and what it is testing. 6 months from now if this fails we'll be really confused